Makefile

```diff
@@ -0,0 +1,23 @@
+base_dir=$(abspath ./)
+
+CHISEL_VERSION=3.6.0
+MODEL ?= TestHarness
+PROJECT ?= freechips.rocketchip.system
+CFG_PROJECT ?= $(PROJECT)
+CONFIG ?= $(CFG_PROJECT).DefaultConfig
+MILL ?= mill
+
+scala_srcs := $(shell find $(base_dir) -name "*.scala" -o -name "*.sc")
+resource_dirs := $(shell find $(base_dir) -type d -path "*/src/main/resources")
+resources := $(foreach d,$(resource_dirs),$(shell find $(d) -type f))
+all_srcs := $(scala_srcs) $(resources)
```

Comment:
No need to maintain these dependencies anymore, since mill already handled the build dependency evaluation.

Reply:
Good point. I will remove